### PR TITLE
Hours-28

### PR DESCRIPTION
**Pull request by barmintor**
* Convert open and close to timestamps
* Permit close time to precede open time to indicate overnight hours, and alert on assignment

### CODE REVIEW

File: spec/features/open_now_spec.rb [hunk 1]
```diff
@@ -1,20 +1,70 @@
 require 'rails_helper'
 
-describe 'locations open now' do
-  context 'when one library closed and one open' do
+describe 'locations open now', type: :feature do
+  let(:now) { Time.now }
+  let(:tomorrow) { now + 1.day }
+  let(:fourAM) { Time.local(tomorrow.year, tomorrow.month, tomorrow.day , 4, 0, 0) }
```
Comment (cgalarza) [inline spec/features/open_now_spec.rb]: This should be `Time.zone.local`. Might be where those travis errors are coming from.

**Review by cgalarza — APPROVED** [review]
👍 Code looks good. I really like the addition of the developer strategy.

I wouldn't change the timezone on the travis machine, because if we use the correct methods we can avoid most problems. It also protects us, if the timezone of our VMs doesn't get set correctly.

**Comment by barmintor** [comment]
My only concern is that Time.local should be the same as Time.zone.local?

**Comment by cgalarza** [comment]
`Time.local` uses the TZ of the machine, while `Time.zone.local` forces the use of the timezone set in the application configuration. `Time.zone.local` is a rails method, while `Time.local` is a ruby method.

**Comment by barmintor** [comment]
Oh, wow. The more you know! I'll revert the travis change and update

**Pull request merged**